app/lib/screens/OnboardingScreens/LogIn.dart

```diff
@@ -42,7 +43,7 @@ class _LoginScreenState extends State<LoginScreen> {
               child: BlocListener<SignInBloc, SignInState>(
                 listener: (context, state) {
                   final formStatus = state.formStatus;
-                  if (formStatus is SubmissionFailed) {
+                  if (formStatus is SubmissionFailed && signInclicked) {
```

Comment:
Can we not just pop the context out for not showing snackbar rather than controlling it with boolean?

Reply:
@sachintha-s what about this?

Reply:
this fix also means, that if it fails the first time and you try again, you won't see the error message of the second. What about trying @gtalha07's way?

Reply:
I don't think its because of bloc listener, it only listen for state changes and doesn't control widgets. In that way it should also display login success snackbar constantly which isn't . Although I've not been encountering this issue on my real android device but if its appearing on other test devices then it's a bug. But I think we might have even better solution for this. Snackbar provides methods for controlling duration. As stated in the docs,
'To control how long a [SnackBar] remains visible, use [SnackBar.duration].'
'To remove the [SnackBar] with an exit animation, use [hideCurrentSnackBar] or call [ScaffoldFeatureController.close] on the returned [ScaffoldFeatureController]. To remove a [SnackBar] suddenly (without an animation), use [removeCurrentSnackBar].'
I suggest we should go with either of these options.